src/assets/sass/protocol/components/_language-switcher.scss

```diff
@@ -12,6 +12,8 @@
     .mzp-c-cta-link {
         @include bidi(((margin, 0 $margin-lg $margin-sm 0, 0 0 $margin-sm $margin-lg),));
         @include text-body-sm;
+        border-bottom: none;
```

Comment:
This selector doesn't appear to be in master anymore since the new footer landed, does this branch need a rebase?